lib/open_mic.rb

```diff
@@ -0,0 +1,30 @@
+class OpenMic
+  attr_reader :char_params, :performers
+
+  def initialize(char_params)
+    @location = char_params[:location]
+    @date = char_params[:date]
+    @performers = []
+  end
+
+  def location
+    @location
+  end
+
+  def date
+    @date
+  end
+
+  def welcome(performer)
+    @performers << performer
+  end
+
+  def repeated_jokes?
```

Comment:
This is working for the scenario in your test, but I'm not sure it will work for all situations.  I think a simpler solution is to compare a list of all jokes, with a uniq list of all jokes:

```
all_jokes = @performers.flat_map { |performer| performer.jokes }

all_jokes.length != all_jokes.uniq.length
```